helm/examples/efs/hpcc-efs/templates/hpcc-efs-rbac.yaml

```diff
@@ -0,0 +1,52 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ printf "%s-rbac" (include "hpcc-efs.fullname" $) }}
+
+rules:
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "watch", "create", "update", "patch"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ printf "%s-rbac" (include "hpcc-efs.fullname" $) }}
```

Comment:
I'm not sure if it's a problem, but now the "Role" above, and this "RoleBinding" both have the same name,
think one should be changed.